requirements.txt

```diff
@@ -1,3 +1,5 @@
 ovos-utils~=0.0, >=0.0.28
 ovos_workshop~=0.0, >=0.0.12a27
-ovos-bus-client
\ No newline at end of file
+ovos-bus-client
+ovos-config
+ovos-core
```

Comment:
ovos-core is not used, skills now should only depend on ovos_workshop

this also allows skills to run fully standalone (see example in ovos-docker)